build.gradle.kts

```diff
@@ -16,4 +19,54 @@ dependencies {
 
 tasks.withType<KotlinCompile> {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tokgozmusa/detekt-danger/blob/188a4025652c68e08e9ac84e526e71fcf9c28a5a/build.gradle.kts#L20"></span>This comment contains text that has been defined as forbidden in detekt.</td>
    </tr>
  </tbody>
</table>